src/introduction/introduction.tex

```diff
@@ -14,33 +14,35 @@ \chapter{Introduction}\label{ch:introduction}
 % textidote: ignore begin
 \begin{figure}[htb]
     \centering
-    \includegraphics[width=1\textwidth]{overview}
-    \caption{The two different overall chess learning environments.}\label{fig:project-overview}
+    \includegraphics[width=0.7\textwidth]{overview}
+    \caption{The two different overall chess learning environments.
+    Traditional coaching is a physical coaching session where a coach and some players meet up and play over the board.
+    Chess engine coaching exists online in the cloud, no physical meeting is necessary.}\label{fig:project-overview}
 \end{figure}
 % textidote: ignore end
 
-Chess engine popularity has risen greatly in the past years and have been developed to a point that they
+Chess engine popularity has risen greatly in the past years and has been developed to a point that they
 outperform humans.
-These tools are essential to understanding the complexity of the end- and mid-game strategies of chess.
-This makes engines an optimal tool for teaching players the intricacies of chess which is what
+These tools are essential to understanding the complexity of mid/end-game strategies of chess.
```

Comment:
```suggestion
These tools are essential to understanding the complexity of mid- and endgame strategies of chess.
```

Reply:
I think that "mid- to endgame" sounds better